modules/_labs/select/react/lib/SelectBase.tsx

```diff
@@ -110,6 +111,21 @@ export interface SelectBaseProps extends CoreSelectBaseProps {
    * @default true
    */
   isMenuAnimated: boolean;
+  /**
+   * If true, automatically flip the SelectBase menu to keep it visible if necessary (e.g., if the the menu would otherwise display below the visible area of the viewport).
+   * @default true
+   */
+  isMenuAutoFlipped: boolean;
+  /**
+   * If true, focus the SelectBase menu when it's shown. Set to false if you don't want to focus the menu automatically (for visual testing purposes, for example).
+   * @default true
+   */
+  isMenuAutoFocused: boolean;
+  /**
+   * If true, flip the SelectBase menu so it extends upwards from the button.
+   * @default false
+   */
+  isMenuFlipped: boolean;
```

Comment:
Are these all config for visual testing? If so, we should consider a way to do static visual things without requiring public props for this purpose. Context is a possibility.

Reply:
That's correct. I'll note this down to tackle later.